signac/__main__.py

```diff
@@ -97,10 +97,6 @@
 warnings.simplefilter("default")
 
 
-def _print_err(msg=None, *args):
```

Comment:
Does it matter that the _print_err is now doing (msg, file=sys.stderr, *args) instead of the original position given?  (more an aesthetic question rather than functionality)

Reply:
The function is now defined as:
```python
def _print_err(*args, **kwargs):
    print(*args, file=sys.stderr, **kwargs)
```
This use of `*args` and `**kwargs` will transparently pass through all positional and keyword arguments, with one exception: we always provide `file=sys.stderr`. If the caller provides a keyword argument for `file`, the function will raise an error. This is okay because we're only using it as a shorthand utility for printing to `stderr`.

The previous function definition did not support other keyword arguments like `print("some", "words", end="")`, which provides an empty `end` value (rather than including a new line at the end of the output). The new definition corrects this behavior, which is needed for this call:
https://github.com/glotzerlab/signac/blob/8303d2e579b2b39621aa8a8cb35bc743a496b0b1/signac/contrib/migration/__init__.py#L113-L116